### PR TITLE
Chore: update style, link modal with app view, call db on button clicks

### PR DESCRIPTION
**Pull request by trevorwhealy**

### CODE REVIEW

**Comment by trevorwhealy** [comment]
closes #36

File: client/AddTask.jsx [hunk 1]
```diff
@@ -46,7 +46,13 @@ class AddTask extends React.Component {
       return;
     }
 
-    this.props.onAddNewTask(taskName, dueDate);
+    socket.emit('create task', {
```
Comment (cassiebaer) [inline client/AddTask.jsx:49]: Move this into a 'frontend' service.. which is the new feature you are working on now

**Comment by cassiebaer** [comment]
:shipit:

**Comment by donovanCarreno** [comment]
:shipit:

**Pull request merged**